bazel/external_crates.bzl

```diff
@@ -234,6 +234,12 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
             "bit-vec": crate.spec(
                 version = "^0.6.3",
             ),
+            "bitcoin-0-32": crate.spec(
```

Comment:
nit: maybe rename to something like `bitcoin-no-default"? I find it a bit weird that if you switch between `bitcoin` and `bitcoin-0-32` as a dependency, then not the same features are included. (I'm aware that my suggestion would make it less clear the a different version is used).